core/domain/classifier_services.py

```diff
@@ -327,17 +311,17 @@ def save_classifier_training_job(algorithm_id, exp_id, exp_version,
             classifier.
 
     Returns:
-        job_id: str. ID of the classifier training job.
+        job_id: str or None. ID of the classifier training job.
     """
-    classifier_training_job_model = (
-        classifier_models.ClassifierTrainingJobModel.get(job_id, False))
-    if classifier_training_job_model is None:
+    if job_id is None:
         classifier_training_job = classifier_domain.ClassifierTrainingJob(
-            'job_id_dummy', algorithm_id, exp_id, exp_version,
+            'job_id_dummy', algorithm_id, interaction_id, exp_id, exp_version,
```

Comment:
@seanlip I thought we agreed on doing this. This domain object is created just for validation purposes. We need to validate the data before storing it. We don't really have any other way to do validate() without a domain object.